lib/rubocop/cop/minitest/file_name.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Minitest
+      # Checks if test file names start with `test_` or end with `_test.rb`.
+      #
+      # @example
+      #   # bad
+      #   my_class.rb
+      #
+      #   # good
+      #   my_class_test.rb
+      #   test_my_class.rb
+      #
+      class FileName < Base
+        include MinitestExplorationHelpers
+
+        MSG = 'Test file path should end with `_test.rb`.'
+
+        def on_class(node)
+          return unless test_class?(node)
+
+          add_offense(node) unless valid_file_name?
+        end
+
+        private
+
+        def valid_file_name?
+          file_name.match?('^test_.*') || file_name.match?('.*_test\.rb$')
```

Comment:
`start_with?`/`end_with?` ?
Do we really need to check for `.rb` file extension? 🤔